### PR TITLE
Update genesis.json

### PR DESCRIPTION
**Pull request by Tomas-Eminger**
Rockaway Blockchain Fund validator for testnet

### CODE REVIEW

File: testnet1/genesis.json [hunk 1]
```diff
@@ -188,7 +197,14 @@
         "ethereum_address": "0xa8CE9eF6ac6C4a1cE18aEd13bA1EFfa718c0e481",
         "info_url": "https://lovali.xyz",
         "country": "FI"
-      }
+      },
+      "Pk0RvTDEjG8h4vH0s2Zk5+TZLAgdJ8HETaovD++76hg=": {
+        "vega_pub_key": "df94148a3c7b8b6d07e4e69ff95ca011405cf0078659decd0d5aab0232b2e4ef",
+        "ethereum_address": "0x4E34DdF693EcCcc184cbc62A825C96942a7F2676",
+        "tm_pub_key": "",
+        "info_url": "https:/rbf.capital/staking",
```
Comment (MM0819) [inline testnet1/genesis.json:205]: Can you fix the URL it looks broken (missing forward slash after HTTPS)

File: testnet1/genesis.json [hunk 2]
```diff
@@ -110,7 +110,16 @@
...
-    }
+    },
+    {
+      "address": "E71A8125505F3D2ACB8CC302EBE0B41F2E87A21D",
+      "pub_key": {
+        "type": "tendermint/PubKeyEd25519",
+        "value": "Pk0RvTDEjG8h4vH0s2Zk5+TZLAgdJ8HETaovD++76hg="
+      },
+      "power": "10",
+      "name": ""
```
Comment (MM0819) [inline testnet1/genesis.json:121]: Can you add your name please

File: testnet1/genesis.json (hunk 1 above)
Comment (Tomas-Eminger) [reply testnet1/genesis.json]: Done

File: testnet1/genesis.json (hunk 2 above)
Comment (Tomas-Eminger) [reply testnet1/genesis.json]: Done

**Pull request merged**